internal/services/mobilenetwork/mobile_network_packet_core_control_plane_resource_test.go

```diff
@@ -0,0 +1,444 @@
+  access_policy {
+    tenant_id               = data.azurerm_client_config.test.tenant_id
+    object_id               = data.azurerm_client_config.test.object_id
+    secret_permissions      = ["Delete", "Get", "Set", "Purge"]
+    certificate_permissions = ["Create", "Delete", "Get", "Import", "Purge"]
+  }
+
+  access_policy {
+    tenant_id               = data.azurerm_client_config.test.tenant_id
+    object_id               = azurerm_user_assigned_identity.test.principal_id
+    secret_permissions      = ["Delete", "Get", "Set"]
+    certificate_permissions = ["Create", "Delete", "Get", "Import"]
+  }
+
+}
+
+resource "azurerm_key_vault_certificate" "test" {
+  name         = "acctest-mn-%[2]d"
+  key_vault_id = azurerm_key_vault.test.id
+
+  certificate {
+    contents = filebase64("testdata/rsa_bundle.pfx")
+    password = ""
+  }
+}
+
+resource "azurerm_mobile_network_packet_core_control_plane" "test" {
+  name                              = "acctest-mnpccp-%[2]d"
+  resource_group_name               = azurerm_resource_group.test.name
+  location                          = "%[3]s"
+  sku                               = "G0"
+  core_network_technology           = "5GC"
+  site_ids                          = [azurerm_mobile_network_site.test.id]
+  control_plane_access_name         = "default-interface"
+  control_plane_access_ipv4_address = "192.168.1.199"
+  control_plane_access_ipv4_gateway = "192.168.1.1"
+  control_plane_access_ipv4_subnet  = "192.168.1.0/25"
+
+  local_diagnostics_access {
+    authentication_type          = "AAD"
+    https_server_certificate_url = azurerm_key_vault_certificate.test.versionless_secret_id
+  }
+
+  identity {
+    type = "UserAssigned"
+    identity_ids = [
+      azurerm_user_assigned_identity.test.id,
+    ]
+  }
+
+  platform {
+    type           = "AKS-HCI"
+    edge_device_id = azurerm_databox_edge_device.test.id
+  }
+
+  depends_on = [azurerm_mobile_network.test]
+}
+
+
+
+
+`, r.template(data), data.RandomInteger, data.Locations.Primary)
+}
+
+func (r MobileNetworkPacketCoreControlPlaneResource) withInteropJson(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+				%s
+
+resource "azurerm_mobile_network_packet_core_control_plane" "test" {
+  name                              = "acctest-mnpccp-%d"
+  resource_group_name               = azurerm_resource_group.test.name
+  location                          = "%s"
+  sku                               = "G0"
+  site_ids                          = [azurerm_mobile_network_site.test.id]
+  control_plane_access_name         = "default-interface"
+  control_plane_access_ipv4_address = "192.168.1.199"
+  control_plane_access_ipv4_gateway = "192.168.1.1"
+  control_plane_access_ipv4_subnet  = "192.168.1.0/25"
+
+  local_diagnostics_access {
+    authentication_type = "AAD"
+  }
+
+  platform {
+    type           = "AKS-HCI"
+    edge_device_id = azurerm_databox_edge_device.test.id
+  }
+
+  interop_json = jsonencode({
+    "mtu" = 1440
+  })
```

Comment:
What fields can be set in here? Is this defined anywhere?

Reply:
This acts like a custom field, when users using it, they should get this from the support (or sales, I'm not sure). And confirm with service team if this could be some fixed properties, they gave a negative answer and suggest let it accept json input... does it make sense?

Reply:
As with other services that choose to only expose a generic JSON to configure a resource this is problematic for obvious reasons, e.g. the introduction of breaking changes, inability to perform any validation on properties etc.

It seems strange that this was allowed into the swagger and if the service team knows what values can be set in this JSON then they should be appropriately documented and specified.

cc. @jackofallops

Reply:
discussed with service team, this is designed to be free JSON.  
"the purpose of the interop settings is to allow third party vendors to supply their own values".

With this could it be exposed as a generic JSON?

Reply:
in which case, can we give this a more descriptive name - `interop_json` is pretty unclear as to what this is, if this is configuration settings, we should say that?

Reply:
is `vendors_interop_json` a good name?

Reply:
make sense, thanks!